pkg/internal/computeresources/transformer_test.go

```diff
@@ -313,7 +313,7 @@ func TestTransformerOneContainer(t *testing.T) {
 			Containers: []corev1.Container{{
 				Resources: corev1.ResourceRequirements{
 					Limits: corev1.ResourceList{
-						corev1.ResourceMemory:           resource.MustParse("4Gi"),
+						corev1.ResourceMemory:           resource.MustParse("2Gi"),
```

Comment:
these updates on previous testcases are to align with updated logics on "setting requests/limits with limitranges"